client/jetpack-cloud/sections/partner-portal/issue-license-form/style.scss

```diff
@@ -29,46 +29,57 @@
 }
 
 .issue-license-form__controls {
-	text-align: right;
-
-	@include break-large {
-		text-align: left;
+	position: fixed;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	padding: 0.875rem 0.875rem 1.5rem;
```

Comment:
Strictly subjective opinion: I would prefer this to have been `padding: 0.875rem` since the extra space at the bottom doesn't seem to be needed on the devices I have tested and my eye feels like it looks off because of the different spacings.

That being said: it looks correct according to the design, so I don't expect any changes here 😄